docs/explanation/security.md

```diff
@@ -29,3 +29,14 @@ If `$PEBBLE_PERSIST` is set to "never", then Pebble will only keep the state in
 ## Security updates
 
 There are several ways to install Pebble. The easiest way to ensure that you get security updates is to [install the snap](#install_pebble_snap).
+
+
+## Cryptographic technology
+
+Pebble uses cryptographic technology for the "basic" [identity](../reference/identities.md) type and for its TLS handling code.
+
+For the basic identity type, Pebble uses Ulrich Drepper's [SHA-crypt algorithm](https://www.akkadia.org/drepper/SHA-crypt.txt) with SHA-512. Specifically, we use the third party Go library [github.com/GehirnInc/crypt](https://github.com/Gehirninc/crypt) for verifying the password hashes sent in a client's `Authorization` HTTP header.
+
+For TLS, Pebble uses the TLS code in Go's standard library when the `--https` argument is passed to `pebble run`, enabling API access over TLS. At present Pebble doesn't do certificate handling, so this only works with `curl --insecure`, and is therefore of limited use. Our intention here is that projects that build on Pebble can [override how TLS connections are verified](https://pkg.go.dev/github.com/canonical/pebble@v1.23.0/client#Config).
```

Comment:
I would tweak the part about "certificate handling" a bit. Here is a quick thought - please alter as you see fit:

```diff
+ new_line_of_code;
```

Reply:
Thanks, done.